features/com.jboss.devstudio.core.feature/feature.xml

```diff
@@ -23,4 +23,7 @@
   <includes id="com.jboss.devstudio.core.rpm.feature" version="0.0.0"/>
   <includes id="com.jboss.devstudio.core.rpmdeps.feature" version="0.0.0"/>
 
+  <!-- features excluded from rpm -->
+  <includes id="org.jboss.tools.jsf.feature" version="0.0.0"/>
```

Comment:
We need jsf feature in PRM too. JSF is much bigger than just VPE extensions. Let's:
1. move all jsf.vpe.* plugins from jsf feature to a new one - jsf.vpe.feature.
2. include vpe feature to jsf.vpe.feature and add it to devstudio.core only.
3. remove vpe feature from jsf feature